### ISSUE TITLE
ui to curate make and model information associated with airplanes

### ISSUE THREAD

**Issue by wbreeze** [wbreeze/iaccdb#51]
part of issue #40. depends upon issue #49 

implement an authorized ui to curate make and model information.

view make and model associated with airplanes and allow alteration of the same

includes a kind of merge function where all airplanes associated with M different mNm combinations can be merged into one selected mNm combination. Something like member merge, but much simpler.
